include/onnxruntime/core/framework/kernel_def_builder.h

```diff
@@ -64,6 +65,10 @@ class KernelDef {
     return alias_map_;
   }
 
+  const nonstd::optional<std::pair<int, int>>& VariadicAlias() const {
```

Comment:
let's use onnxruntime::optional instead of the nonstd::optional directly - the idea is maybe someday onnxruntime::optional can refer to std::optional

Reply:
ok